core/src/main/java/hudson/model/Queue.java

```diff
@@ -1521,7 +1521,12 @@ public Api getApi() {
      * also represents the "primary" sub-task (and as implied by this
      * design, a {@link Task} must have at least one sub-task.)
      * Most of the time, the primary subtask is the only sub task.
+     * 
+     * <p>
+     * Tasks are being exported to API in {@link Item#task}, so every child class
+     * has an {@link ExportedBean} attribute.
      */
+    @ExportedBean
     public interface Task extends ModelObject, SubTask {
```

Comment:
This will have no effect, I think—did you test it somehow? `@ExportedBean` is `@Inherited` but that meta-annotation only takes effect on superclasses, not implemented interfaces.